### ISSUE TITLE
Few accessibility issues when using MISP through a screen reader

### ISSUE THREAD

**Issue by obert01** [MISP/MISP#2038]
Hello,

Because I am visually impaired (actually nearly blind), I am using MISP with a screen reader (JAWS and NVDA generally). 

My problem is that various pages include what's called "clickable text", which cannot be activated because they are not properly declared in regard of ARIA standards. "clickable text" are "span" elements on which an "onclick" action is defined. 

For example, when viewing an event, in the list of attributes, the "propose enrichment", "add enrichement", "propose deletion" and "delete" buttons are not accessible for a screen reader. 

In the following final html code for example, the buttons are not accessible:
```
<td class="short action-links ">
<span class="icon-asterisk useCursorPointer" onClick="simplePopup('https://misp-backend.cert-ist.com/events/queryEnrichment/478235/ShadowAttribute');" title="Propose enrichment">&nbsp;</span>
...
<span class="icon-trash useCursorPointer" title="Propose Deletion" onClick="deleteObject('shadow_attributes', 'delete', '478235', '3893');"></span>
<span class="verticalSeparator">&nbsp;</span>
<span class="icon-asterisk useCursorPointer" onClick="simplePopup('https://misp-backend.cert-ist.com/events/queryEnrichment/478235/Attribute');" title="Add enrichment">&nbsp;</span>
...
<span class="icon-trash useCursorPointer" onClick="deleteObject('attributes', 'delete', '478235', '3893');"></span>
</td>
```

The solution is to add the following properties to the "span" elements:
```
role="button" tabindex="0" aria-label="blablabla"
```

Note: The problem is present in various pages of the MISP interface. 

ARIA tips for making accessible buttons are nicely explained here:
https://www.marcozehe.de/2013/04/24/easy-aria-tip-6-making-clickables-accessible/

Thank you in advance for your support.

**Comment by adulau** [comment]
Very good point, thank you very much for your feedback. We need to fix this asap.

**Comment by iglocska** [comment]
Thanks a lot for letting us know, I've gone through most of the views and added the missing attributes to the span fields that serve as buttons, but I might have missed some things - let us know if anything doesn't work as expected!

**Comment by obert01** [comment]
What a quick and good job ! Thank you !

For the moment, I have not encountered any unproperly declared button.

**Comment by iglocska** [comment]
Excellent, keep us posted if you run into issues and thanks for the guidance on how to fix it!

**Comment by obert01** [comment]
Hello,

There remain an issue in the ajax import and export popup windows. 

For example, in the ajax "populate from ..." import window, all buttons are now labelled "import", they can't be distinguished. Apparently, this is due to the "aria-label" property being improperly set. 

Thank you very much.

**Comment by iglocska** [comment]
Thanks for the heads-up, will fix it asap!

**Comment by iglocska** [comment]
Should be fixed now!
